### PR TITLE
mariadb-connector-c 3.0.1 (new formula)

### PR DESCRIPTION
**Pull request by denji**

### CODE REVIEW

File: Library/Formula/mariadb-connector-c.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+class MariadbClientRequirement < Requirement
+  fatal true
+  default_formula "mariadb"
+  satisfy { which "mysql" }
+end
```
Comment (mikemcquaid) [inline Library/Formula/mariadb-connector-c.rb]: Just use `:mysql` which also does `which "mysql"`

File: Library/Formula/mariadb-connector-c.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+class MariadbConnectorC < Formula
+  desc "MySQL & MariaDB database connector for C/C++ applications"
+  homepage "https://github.com/MariaDB/mariadb-connector-c"
+  stable do
+    url "https://github.com/MariaDB/mariadb-connector-c.git",
+        :revision => "330b7fbdbabad44180862316d8925b302d1a8409"
```
Comment (mikemcquaid) [inline Library/Formula/mariadb-connector-c.rb]: Use a tagged version.

File: Library/Formula/mariadb-connector-c.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    version "3.0.0"
+    # link _libiconv_* symbols on OS X
+    # https://github.com/MariaDB/mariadb-connector-c/issues/4
+    patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/mariadb-connector-c.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

File: Library/Formula/mariadb-connector-c.rb (hunk 2 above)
Comment (denji) [inline Library/Formula/mariadb-connector-c.rb]: @mikemcquaid https://github.com/MariaDB/mariadb-connector-c/tags ?

Comment (DomT4) [inline Library/Formula/mariadb-connector-c.rb]: Ask for one.

File: Library/Formula/mariadb-connector-c.rb (hunk 1 above)
Comment (denji) [inline Library/Formula/mariadb-connector-c.rb]: @mikemcquaid `:mysql => "mariadb"` consumer preference `default_formula`? 

`:postgresql => "postgresql94"`

File: Library/Formula/mariadb-connector-c.rb (hunk 2 above)
Comment (denji) [inline Library/Formula/mariadb-connector-c.rb]: @DomT4 Release commit https://github.com/MariaDB/mariadb-connector-c/commit/330b7fbdbabad44180862316d8925b302d1a8409

Comment (denji) [inline Library/Formula/mariadb-connector-c.rb]: @DomT4 These guys don't understand how to use git, release commit: https://github.com/MariaDB/mariadb-connector-c/commit/330b7fbdbabad44180862316d8925b302d1a8409
/cc @9E0R9

Comment (mikemcquaid) [inline Library/Formula/mariadb-connector-c.rb]: Please ask them for a tagged release, thanks.

File: Library/Formula/mariadb-connector-c.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/mysql_config", "--cflags"
+    system "#{bin}/mysql_config", "--include"
+    system "#{bin}/mysql_config", "--libs"
+    system "#{bin}/mysql_config", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/mariadb-connector-c.rb]: Only need one of these.

**Comment by bfontaine** [comment]
Ping?

**Pull request closed without merging**